tests/test_store.py

```diff
@@ -90,9 +90,15 @@ def test_integrity_error_returns_500_feedback(self):
         db.drop_all()
 
     # /responses/<tx_id> GET
+    def test_get_id_returns_400_if_not_a_valid_uuid(self):
+        """Endpoint should return 400 if the tx_id isn't a valid uuid formatted uuid"""
+        r = self.app.get(self.endpoints['responses'] + '/123')
+        assert r.status_code == 400
```

Comment:
Worth adding an additional test covering the case that was observed in production? i.e. a mostly valid looking UUID with a \n ^\n at the end?

Reply:
@samiwel Done